dojo/tools/h1/parser.py

```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
         # Load the contents of the JSON file into a dictionary
         data = file.read()
         try:
-            tree = json.loads(str(data, 'utf-8'))
-        except:
+            tree = json.loads(str(data, "utf-8"))
+        except BaseException:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50